src/lwpCall.js

```diff
@@ -193,17 +193,19 @@ export default class {
     }
   }
 
-  mute(options = {}) {
-    options = lwpUtils.merge(options, { audio: true, video: true });
-
+  /**
+   * @param {{audio: boolean, video: boolean}} options - The channels you want to mute
+   */
+  mute(options = { audio: true, video: true }) {

```

Comment:
My only concern is this part,  I think mute audio and video should be set as false by default to keep backward compatibility, otherwise, if you want to migrate from an earlier version you must add that mute option on the initial Libwebphone configuration. The less code you have to add for migration, the better.

Reply:
Oh, so the way this thing works is confusing, we are just declaring which channels are going to be muted. This function still has the same default values as before. I think you might be referring to[ these lines](https://github.com/2600hz/libwebphone/pull/34/files#diff-9c5e02d0c18fecc696cb0a62fb332990550eeb0bbe19f9d818dd2d87df07e108R127-R128)

Reply:
Yes, I meant those lines you referred to. I just want to emphasize that before these changes the audio and video are not muted by default and now with these changes they are. In other words, you have to add `startWithAudioMuted: false`  and  `startWithVideoMuted: false` explicitly to keep the backward compatibility. I'm not sure if that is a big deal or not.

Reply:
I would think not, as users would need to update their dependencies for these changes to take place. But maybe someone else has any suggestions.

Reply:
I agree with @Irvandoval, it is better to keep backwards compatibility. But if it is this library is not public yet, or it is not highly used, it would be a good time to make a decision on this. From my side, I would keep the same defaults as JSSip.

Reply:
Fixed. Thanks!